lib/jwt/signature.rb

```diff
@@ -0,0 +1,105 @@
+# frozen_string_literal: true
+require 'openssl'
+
+module JWT
+  module Signature
+    extend self
+
+    HMAC_ALGORITHMS = %w(HS256 HS384 HS512).freeze
+    RSA_ALGORITHMS = %w(RS256 RS384 RS512).freeze
+    ECDSA_ALGORITHMS = %w(ES256 ES384 ES512).freeze
+
+    NAMED_CURVES = {
```

Comment:
This hash has been moved into the JWT::DefaultOptions module. Please merge the current master into this PR.